app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -1088,6 +1092,17 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
             return
         isShowingError = true
 
+        // If the endpoint requires a TLS client certificate and we got a timeout, ignore the error
+        // It is likely due to the user not choosing a key yet
+        val tlsWebViewclient = webView.webViewClient as TLSWebViewClient
+        if (tlsWebViewclient != null &&
```

Comment:
`tlsWebViewclient` is not nullable, so there's no need to do this null-check.

Reply:
:+1: